src/Sarif.Converters.UnitTests/FxCopConverterTests.cs

```diff
@@ -328,6 +328,15 @@ public void FxCopLogReader_Context_RefineProjectToExceptionIssue()
 
     public class FxCopConverterTests
     {
+        private static void ValidateLogicalLocations(IList<LogicalLocation> expectedLogicalLocations, IList<LogicalLocation> actualLogicalLocations)
+        {
+            for (int i = 0; i < expectedLogicalLocations.Count; i++)
+            {
+                expectedLogicalLocations[i].ValueEquals(actualLogicalLocations[i]).Should().BeTrue();
+            }
+            actualLogicalLocations.Count.Should().Be(expectedLogicalLocations.Count);
```

Comment:
>actualLogicalLocations [](start = 12, length = 22)

Consider: validating the count first. If the count doesn't match there's no reason to compare the elements.